install/0000_30_machine-api-operator_00_credentials-request.yaml

```diff
@@ -82,8 +82,46 @@ spec:
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: AzureProviderSpec
-    roleBindings:
-    - role: Contributor
+    permissions:
+    - Microsoft.Compute/availabilitySets/delete
+    - Microsoft.Compute/availabilitySets/read
+    - Microsoft.Compute/availabilitySets/write
+    - Microsoft.Compute/disks/delete
```

Comment:
Based on the comments [here](https://github.com/openshift/machine-api-provider-azure/blob/97ea1221e4ca999a161ddee4d1e075880fcbad67/pkg/cloud/azure/services/disks/disks.go#L34), I have only added the permission for `disks/delete`.  I'm not sure yet whether the `read` and `write` permissions are required.